src/screens/Home/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import Background from '../../components/basics/layout';
```

Comment:
I'm not sure but can we use `'components/basics/layout'` here?

Reply:
I think we can set it in the configuration somehow 🤔

Reply:
we can configure in `jsconfig.json`. More detail can be found in this [blog](https://medium.com/hackernoon/absolute-imports-with-create-react-app-4c6cfb66c35d)